apps/teachers/src/styles/globals.css

```
@@ -0,0 +1,1248 @@
+    min-height: 700px;
+    margin: 0;
+    padding: 20px;
+    font-family: sans-serif;
+    text-align: center;
+    background-color: paleturquoise;
+  }
+}
+
+/* FONT IMPORTS */
+
+@import url('https://fonts.googleapis.com/css?family=Open+Sans:300,400,700');
```

Biome (1.9.4)

[error] This @import is in the wrong position.

Any @import rules must precede all other valid at-rules and style rules in a stylesheet (ignoring @charset and @layer), or else the @import rule is invalid.
Consider moving import position.

(lint/correctness/noInvalidPositionAtImportRule)